Code/Framework/AzToolsFramework/AzToolsFramework/Viewport/LocalViewBookmarkLoader.cpp

```diff
@@ -413,102 +419,94 @@ namespace AzToolsFramework
             return false;
         }
 
-        // if the field is not empty then we have a file linked to the prefab.
+        // record an undo step if a local view bookmark component is added and configured
+        ScopedUndoBatch undoBatch("SetupLocalViewBookmarks");
+        undoBatch.MarkEntityDirty(bookmarkComponent->GetEntityId());
+
+        // initialize default locations to 0. This is a temporary solution to match the 12 locations of the legacy system
+        // once there is a UI for the view bookmarks these lines should be removed
+        auto createDefaultBookmarksFn = [this, &registry]()
```

Comment:
nit: This is only used in the return statement at the end of the function, moving its declaration at the end just before it's used would be nicer.

Reply:
I've updated where this logic lives now to make it clearer, this is now updated 👍